package.json

```diff
@@ -148,8 +148,8 @@
     "trezor-connect": "6.0.2",
     "validator": "6.3.0"
   },
-  "engineStrict": true,
-  "engine": {
+  "engine-strict": true,
```

Comment:
IS this a valid key in `package.json`? Googling seems to indicate this has to be in `npmrc`

Reply:
ohh yes, will delete `"engine-strict": true,` key and in `.npmrc` 

work very well:
```
shin@u18-14-01:~/repo/emurgo/yoroi-frontend$ npm install
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for yoroi@1.2.1: wanted: {"node":"~8.9.4","npm":"~5.6.0"} (current: {"node":"10.15.1","npm":"6.4.1"})
npm ERR! notsup Not compatible with your version of node/npm: yoroi@1.2.1
npm ERR! notsup Not compatible with your version of node/npm: yoroi@1.2.1
npm ERR! notsup Required: {"node":"~8.9.4","npm":"~5.6.0"}
npm ERR! notsup Actual:   {"npm":"6.4.1","node":"10.15.1"}

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/shin/.npm/_logs/2019-02-12T15_51_57_621Z-debug.log
shin@u18-14-01:~/repo/emurgo/yoroi-frontend$ 
```